.identity/03_managed_identity_cd.tf

```diff
@@ -0,0 +1,18 @@
+module "identity_cd" {
+  source = "github.com/pagopa/terraform-azurerm-v3//github_federated_identity?ref=v7.34.0"
```

Comment:
Get the latest version

Reply:
done in [42a1bd0](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/42a1bd07a2f1b3618634d703ecbf89c979516510)